examples/conformance/CMakeLists.txt

```diff
@@ -883,6 +1172,201 @@ add_test(NAME "${TS_NAME}_conversion_fast_double_ulong"
 add_test(NAME "${TS_NAME}_conversion_fast_long_double"
          COMMAND "${TS_BUILDDIR}/test_conformance/conversions/test_conversions" -m -4 long_double long_sat_double long_rte_double long_sat_rte_double long_rtp_double long_sat_rtp_double long_rtn_double long_sat_rtn_double long_rtz_double long_sat_rtz_double )
 
+#***************************************************************************
+
+# tests math on vectors of 4 floats.
+# This is usually the fastest version on machines with AVX(2)
+
+#add_test(NAME "${TS_NAME}_math_fast_add"
```

Comment:
What are the commented out tests?

Reply:
ah, forgotten. i'll uncomment them

Reply:
Do you mean they are not required by the conformance or are already included in some other suite?